<launch>

  <!-- Start image_republisher_node -->
  <node pkg="pepper_local_republisher" type="image_republisher_node" name="pepper_rgb_republisher_node" output="screen">
    <param name="input_image_topic_name"    value="/pepper_robot/camera/front/image_rect_color" />
    <param name="output_image_topic_name"   value="/pepper_local_republisher/pepper_robot/camera/front/image_rect_color" />
  </node>

</launch>
